CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+X.Y.Z Release notes
+=============================================================
+### Compatibility
+* Sync protocol: 24
+* Server-side history format: 4
+* File format: 7
+* Realm Object Server: 3.0.0 or later
+
+### Breaking changes
+* None.
+
+### Enhancements
+* None.
+
+### Bug fixes
+* Fixed a bug if `fullSynchronized` is not defined. This could lead to an exception when opening a synced Realm.
```

Comment:
@kneth Please specify where "fullSynchonized" should be defined. And which version was this introduced?

Reply:
And also mention which version the bug was introduced in and include a reference to the issue/PR number.

Reply:
See https://github.com/realm/realm-js/pull/1998